docs/welcome-to-civicactions/training/intro-open-source.md

```diff
@@ -1,20 +1,20 @@
 # Open source
 
-### What is open source software?
+## What is open source software?
 
-//Something about Drupal here...
+@todo: //Something about Drupal here...
 
 Broadly speaking, open source software is software whose inner workings are made fully public and distributed freely to the community. As an open source team, the software CivicActions writes is made public (mostly at <https://github.com/civicactions>), and we welcome anyone to use that software, copy it, and change it.
 
 Generally (and at CivicActions), people publishing open source software tend to be open to receiving public contributions. So, members of the public may copy our software, modify it, and then submit their modifications back to us for potential inclusion in the original.
 
-### What does it mean in our work?
+## What does it mean in our work?
 
-//
+@todo: //
```

Comment:
Minor quibble, but the @todo's look best when they are in on their own line, blockquotes, like so:
>  @todo blah blah blah